tests/conftest.py

```diff
@@ -2665,9 +2668,122 @@ def cloud_data_context_with_datasource_pandas_engine(
     return context
 
 
+JSONData = Union[AnyPayload, Dict[str, Any]]
```

Comment:
Most of these changes are moving fixtures to a more general conftest.py since they are now used in several test files.